src/Sylius/Bundle/AdminBundle/Resources/views/Order/Label/ShippingState/partially_shipped.html.twig

```diff
@@ -0,0 +1,4 @@
+<span class="ui yellow{% if attached is defined and attached == true %} top attached{% endif %} label">
```

Comment:
I know, I'm annoying and this is irrelevant, but `cancelled` state is yellow 😆

Reply:
Yeah, that's the same with partially paid and cancelled, which already exists for a while and nobody's complaining :D